VersionControlUI/src/DiffManager.cpp

```diff
@@ -196,6 +196,7 @@ bool DiffManager::findChangedNode(Model::TreeManager* treeManager, Model::NodeId
 void DiffManager::createOverlaysForChanges(Visualization::ViewItem* diffViewItem,
 														 QList<ChangeWithNodes> changesWithNodes)
 {
+	static const QString arrowLayer = "move_arrows";
```

Comment:
Err, but now you define this string twice, on this line and on the next one. Remove the next line.